ncm-metaconfig/src/main/pan/components/metaconfig/config-common.pan

```diff
@@ -1,17 +0,0 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
-
-unique template components/${project.artifactId}/config-common;
-
-include { 'components/${project.artifactId}/schema' };
-
-# Set prefix to root of component configuration.
-prefix '/software/components/${project.artifactId}';
-
-#'version' = '${project.version}';
-#'package' = 'NCM::Component';
```

Comment:
Just trying to understand if we should be putting this field in during clean up or taking it out. :-) Can we have a definitive statement please?

Sorry for the confusing statement: it must be ncm-cdispd which triggers a dispatch of a component if /software/packages/ncm-$component/ changes.